website/screens/components/footer/usage/examples/Icons.jsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
```

Comment:
Rename to icons.tsx.
Delete unnecessary import.

Reply:
As for .ts files are in lowercase and .tsx files are with capital letters, I will keep this file in capital letters.
Changed to .tsx.